src/components/Chart.tsx

```diff
@@ -31,11 +33,22 @@ const Chart: React.FC<ChartProps> = (props: ChartProps) => {
     props.onChange(props.chartID, !isSelected);
   };
 
+  const onPrintClick = (event: React.MouseEvent) => {
+    event.stopPropagation();
+    props.history.push(`/PrintCards?chartID=${props.chartID}`);
+  };
+
   return (
     <Card
       className="Chart-Card"
       onClick={() => props.onCardClick(props.chartID)}
     >
+      <img
+        className="Printbox"
+        src={printbox}
```

Comment:
Probably need to find a better icon for the print button. A simple outline of a printer (preferable the same colour and thickness as the checkbox). I'll try look for some too

Reply:
I've updated the image to this while we find one with darker lines
![image](https://user-images.githubusercontent.com/48304096/95072536-d5819f80-0767-11eb-9374-7fce8a609749.png)

Reply:
Can you also add a bit of right margin to the print icon. It's too close to the checkbox, 5px or so would do.